python/paddle/fluid/executor.py

```diff
@@ -626,6 +627,24 @@ def run(self,
 
             list(numpy.array): fetch result according to fetch_list.
         """
+        try:
+            return self._run_impl(
+                program=program,
+                feed=feed,
+                fetch_list=fetch_list,
+                feed_var_name=feed_var_name,
+                fetch_var_name=fetch_var_name,
+                scope=scope,
+                return_numpy=return_numpy,
+                use_program_cache=use_program_cache)
+        except Exception as e:
+            if isinstance(e, core.EOFException):
+                raise e
+            print("An exception was thrown!\n {}".format(str(e)))
+            sys.exit(-1)
```

Comment:
Some user use `try ... except` to catch the  `EOFException`, but `exe.run` may throw other `Exception`.
```
   try:
       loss_data, = exe.run(compiled_prog,
                     fetch_list=[loss.name])
       print(loss_data)
   except Exception as e:
       print("EOF ")
```